usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -514,12 +514,16 @@ def get_or_create_object_class(row_object_class, row_direct_reimbursable, logger
     return obj_class
 
 
-def get_or_create_program_activity(program_activity_code):
+def get_or_create_program_activity(row, submission_attributes):
     # We do it this way rather than .get_or_create because we do not want to
     # duplicate existing pk's with null values
-    prg_activity = RefProgramActivity.objects.filter(ref_program_activity_id=program_activity_code).first()
-    if prg_activity is None and program_activity_code is not None:
-        prg_activity = RefProgramActivity.objects.create(ref_program_activity_id=program_activity_code)
```

Comment:
OK, so... 

Lots of the values in the `submissions` table have 4-character `program_activity_code`s that will fit into `RefProgramActivity.program_activity_code`, but lots of others have 12-character codes.  Should we expand `RefProgramActivity.program_activity_code` to 12 characters (or, perhaps better, simple `TEXT`), slice a 4-character portion of `program_activity_code` (which portion?), or... ?